tests/utils/focus/getActiveElement.ts

```diff
@@ -0,0 +1,32 @@
+import '../../_helpers/shadow-input'
+import {setup} from '#testHelpers'
+import {getActiveElement, getActiveElementOrBody} from '#src/utils'
+
+describe('focus tests with not shadow root', () => {
```

Comment:
Unnecessary describe block. Not all edge cases are necessarily in the binary `with shadow` vs `without shadow` so this might create a legacy of misleading structure.

Reply:
Removed it.